templates/webhook/mutating-webhook-configuration.yaml

```diff
@@ -3,7 +3,7 @@ kind: MutatingWebhookConfiguration
 metadata:
   annotations:
     cert-manager.io/inject-ca-from: kubediag/serving-cert
-  name: mutating-webhook-configuration
+  name: webhook-configuration
```

Comment:
Name of cluster scope resource should be unique, such as `kubediag-mutating-webhook-configuration`